Code/Editor/AzAssetBrowser/AzAssetBrowserRequestHandler.cpp

```diff
@@ -372,6 +372,76 @@ void AzAssetBrowserRequestHandler::AddContextMenuActions(QWidget* caller, QMenu*
             AzToolsFramework::Layers::EditorLayerComponent::CreateLayerAssetContextMenu(menu, fullFilePath, levelPath);
         }
 
+        // Create the callback to pass to the SourceControlAPI
+        AzToolsFramework::SourceControlResponseCallback callback =
+            [fullFilePath](bool success, const AzToolsFramework::SourceControlFileInfo& info)
+        {
+            if (success)
+            {
+                QMessageBox::information(nullptr, "Asset deleted", "Asset name %s", fullFilePath.c_str());
```

Comment:
Use the `AzToolsFramework::GetActiveWindow()` helper for the parent instead of `nullptr`

Same in the below usages as well